### PR TITLE
Add option to set clock at bottom right. AmPm fix.

### PR DESCRIPTION
**Pull request by slackhead**
Hi

I added these some time ago but didn't get around to making a PR.

I added an option to show the clock bottom right. I used -B orginally, but I see you have added bold for that, so I chose -z.

### CODE REVIEW

File: ttyclock.c.orig [hunk 1]
```diff
@@ -0,0 +1,682 @@
+/*
```
Comment (anarcat) [inline ttyclock.c.orig]: this .orig file shouldn't show up here

File: README [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+    -T tty        Display the clock on the specified terminal
+    -r            Do rebound the clock
+    -f format     Set the date format
+    -n            Don't quit on keypress
+    -v            Show tty-clock version
+    -i            Show some info about tty-clock
+    -h            Show this page
+    -D            Hide date
+    -B            Enable blinking colon
+    -d delay      Set the delay between two redraws of the clock. Default 1s.
```
Comment (anarcat) [inline README]: this diff is weird - it shouldn't touch all those lines, only add one.

File: ttyclock.c.orig (hunk 1 above)
Comment (slackhead) [reply ttyclock.c.orig]: Ah must have been left over from a patch that didn't apply.

File: README (hunk 2 above)
Comment (slackhead) [reply README]: Not sure. My original pacthes wouldn't work so I had to add some lines manually. I'll have a look at it later and fix all I can find.

**Comment by slackhead** [comment]
I like the idea of mulitple positions. Maybe a -p 0-8?

**Comment by anarcat** [comment]
On 2018-03-27 16:32:20, Slackhead wrote:
> I like the idea of mulitple positions. Maybe a -p 0-8?

sure, sounds reasonable.

**Comment by slackhead** [comment]
Well the only thing this PR has apart from the bottom position is the am/pm fix on line 195:
ttyclock->meridiem = ((ihour >= 12) ? PMSIGN : AMSIGN);

It's hardly worth a merge just for the addition of a single '='. If you want to add that manually I'll delete this branch and have a look at doing some positioning, unless you want to look at it?

**Comment by anarcat** [comment]
please send the AM/PM fix separately, that one is an easy merge so we can fast-track it apart from this more complicated issue.

**Comment by slackhead** [comment]
Ok I'll delete this branch, repull and start from scratch.

**Comment by anarcat** [comment]
great, thanks for the attention to details! :)

**Comment by slackhead** [comment]
I've just opened the .c in vim and vim is telling me that there is trailing whitespace and mixed tabbed/spaced indents.

I can fix these in a PR. The trailing whitespace is an easy fix, but would you prefer to indent with tabs or spaces, or don't care? Most of it uses spaces, so converting the tabs would be the easiest fix.

**Comment by anarcat** [comment]
On 2018-03-27 18:31:58, Slackhead wrote:
> I've just opened the .c in vim and vim is telling me that there is trailing whitespace and mixed tabbed/spaced indents.
>
> I can fix these in a PR. The trailing whitespace is an easy fix, but would you prefer to indent with tabs or spaces, or don't care? Most of it uses spaces, so converting the tabs would be the easiest fix.

I would prefer keeping whitespace changes separate from other changes,
if you do not mind, because it makes reviews easier that way.

-- 
We have no friends but the mountains.
                        - Kurdish saying

**Comment by slackhead** [comment]
Done.

**Pull request closed without merging**